src/actinia_core/resources/mapset_management.py

```diff
@@ -716,3 +720,49 @@ def _execute(self):
             self.lock_interface.unlock(self.target_mapset_lock_id)
             self.finish_message = \
                 "Mapset <%s> successfully unlocked" % self.target_mapset_name
+
+
+class MapsetLockManagementResourceAdmin(ResourceBase):
+    """ Get all locked mapsets
+    """
+    decorators = [log_api_call, check_user_permissions,
+                  very_admin_role, auth.login_required]
+
+    @swagger.doc({
+        'tags': ['Mapset Management'],
+        'description': 'Get all locked mapsets. '
+                       'Minimum required user role: admin.',
+        'responses': {
+            '200': {
+                'description': 'Get a list of locked mapsets ',
+                'schema': LockedMapsetListResponseModel
+            },
+            '500': {
+                'description': 'The error message and a detailed error log',
+                'schema': SimpleResponseModel
+            }
+        }
+    })
+    def get(self):
+        redis_interface = RedisBaseInterface()
+        kwargs = dict()
+        kwargs["host"] = global_config.REDIS_SERVER_URL
+        kwargs["port"] = global_config.REDIS_SERVER_PORT
+        if global_config.REDIS_SERVER_PW and global_config.REDIS_SERVER_PW is not None:
+            kwargs["password"] = global_config.REDIS_SERVER_PW
+        redis_interface.connect(**kwargs)
```

Comment:
It might be good to disconnect after receiving the needed information from redis. Another possibility would be to use a `with` statement. I am not sure if this is really needed. Other pieces of code here don't explicitly disconnect but reuse existing connections to redis. The ephemeral processing connects as well without disconnecting but as it is called differently, it might behave differently. I am not sure but I guess it won't hurt either.